deploy/chart/templates/22-packageserver.yaml

```diff
@@ -0,0 +1,157 @@
+{{- $ca := genCA "package-server-ca" 365 -}}
+apiVersion: apiregistration.k8s.io/v1beta1
+kind: APIService
+metadata:
+  name: v1alpha1.packages.apps.redhat.com
+spec:
+  caBundle: {{ $ca.Cert | b64enc | quote }}
+  group: packages.apps.redhat.com
```

Comment:
This seems weird since the other group is `operators.coreos.com`

Reply:
I was wondering whether we want to continue using `*.coreos.com`. Also, I added the `package` prefix because the package-server is only going to be responsible for package related resources. If I had just used `apps.redhat.com` or `apps.coreos.com`, the group would be unusable by any other CRDs or extension-apiservers.